jupyterlab/__init__.py

```diff
@@ -45,9 +61,13 @@ def _jupyter_server_extension_paths():
     return [{
         "module": "jupyterlab"
     }]
-    
+
 
 def load_jupyter_server_extension(nbapp):
+    base_dir = os.path.realpath(os.path.join(HERE, '..'))
+    dev_mode = os.path.exists(os.path.join(base_dir, '.git'))
+    if dev_mode:
+        nbapp.log.info(DEV_NOTE_NPM)
     nbapp.log.info('Pre-alpha version of JupyterLab extension loaded from %s'%HERE)
```

Comment:
Can you fix this to say 'JupyterLab alpha preview extension ...' while you're editing this code?